app/js/views/reminders.jsx

```diff
@@ -31,45 +33,56 @@ export default class Reminders extends React.Component {
         this.setState({ reminders });
       });
 
-    this.speechController.on('wakelistenstart', () => {
-      console.log('wakelistenstart');
-    });
-    this.speechController.on('wakelistenstop', () => {
-      console.log('wakelistenstop');
-    });
-    this.speechController.on('wakeheard', () => {
-      console.log('wakeheard');
-    });
-    this.speechController.on('speechrecognitionstart', () => {
-      console.log('speechrecognitionstart');
-    });
-    this.speechController.on('speechrecognitionstop', () => {
-      console.log('speechrecognitionstop');
-    });
-    this.speechController.on('reminder', (reminder) => {
-      console.log('reminder', reminder);
-      const reminders = this.state.reminders;
-      reminders.push({
-        id: reminders.length,
-        recipient: reminder.users,
-        content: reminder.action,
-        datetime: reminder.time,
-      });
+    this.speechController.on('wakelistenstart', this.debugEvent);
+    this.speechController.on('wakelistenstop', this.debugEvent);
+    this.speechController.on('wakeheard', this.debugEvent);
+    this.speechController.on('speechrecognitionstart', this.debugEvent);
+    this.speechController.on('speechrecognitionstop', this.debugEvent);
+    this.speechController.on('reminder', this.onReminder);
```

Comment:
Maybe we could do:
```JS
this.speechController.on('reminder', this.onReminder);
this.speechController.on('reminder', this.debugEvent);
```

We don't have to call `debugEvent` in `onReminder` then.

Would `this` need to be bound to each of these handlers too (at least `onReminder`)?